runscanner/src/main/java/uk/ac/bbsrc/tgac/miso/runscanner/processors/DefaultIllumina.java

```diff
@@ -64,16 +75,19 @@ public static DefaultIllumina create(Builder builder, ObjectNode parameters) {
       case "CompletedAsPlanned":
         return Optional.of(HealthType.Completed);
       default:
-        log.debug("New Illumina completion status found: " + status);
+        log.debug("New Illumina completion status found: %s", status);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Use the built-in formatting to construct this argument. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2629)